docs/platform/howto/restrict-access.md

```diff
@@ -1,18 +1,27 @@
 ---
 title: Restrict network access to services
+sidebar_label: Restrict access
 ---
 
 import ActionsIcon from "@site/static/images/icons/more.svg";
 
 By default, Aiven services are publicly accessible, but you can restrict access to your service to a single IP, an address block, or any combination of both.
 
 :::important
-IP filters apply to publicly accessible endpoints only.
+IP filters apply to inbound traffic coming from specified IP addresses / network ranges to
+Aiven service ports.
 :::
 
+:::Tip
```

Comment:
Let's stay as close to the [docusaurus docs](https://docusaurus.io/docs/markdown-features/admonitions) as possible, including casing, please 🙂 
```suggestion
:::tip
```

(applies to the whole PR, no just this file)